components/dashboard/src/start/StartPage.tsx

```diff
@@ -75,7 +78,7 @@ export interface StartWorkspaceError {
 
 export function StartPage(props: StartPageProps) {
   const { phase, error } = props;
-  let title = props.title || getPhaseTitle(phase, !!error);
+  let title = props.title || getPhaseTitle(phase, error);
   return <div className="w-screen h-screen bg-white align-middle">
     <div className="flex flex-col mx-auto items-center text-center h-screen">
       <div className="h-1/3"></div>
```

Comment:
While you are at it, could you change the height of the placeholder to `h-2/5`,  please? The content sits a bit too high IMHO. 
CC @gtsiolis

Reply:
`h-2/5` looks good, let's do it! ✔️

Reply:
@svenefftinge I think this would push down build logs, which already don't have much vertical space.

Reply:
I think this will push down build logs which already don't have much vertical space.

Reply:
Will try & post screenshots.

Reply:
Ah, yes! Build log output was one of the main reasons we stuck with `h-1/3`.

Reply:
Thanks for confirming! I agree, let's stick with `h-1/3` for now.